### ISSUE TITLE
save_screenshot command in CommandSequence saves a white image

### ISSUE THREAD

**Issue by johanmazel** [citp/OpenWPM#98]
When one uses the save_screenshot command in CommandSequence a white image is saved to the specified directory.

This bug is present in the last (as of now) [commit](https://github.com/citp/OpenWPM/commit/3a14416c5721ac68db7f48803bafa19551dbc70c).

I wrote a [Python script](https://github.com/citp/OpenWPM/files/555262/html_png_test.zip) that generates such image.

I use Ubuntu 16.04.
I checked that the installed selenium version is 2.53.

**Comment by englehardt** [comment]
@dreisman would you mind to look into this?

**Comment by dreisman** [comment]
The problem is you are calling save_screenshot() (and dump_page_source will actually have a similar problem) after dump_profile_cookies(), which closes the current tab in order to reliably save the cookies. It's somewhat obliquely referenced that this is a side effect in our wiki page for commands (https://github.com/citp/OpenWPM/wiki/Available-Commands), but obviously we should comment more and make that clearer.

Basically, dump_profile_cookies and dump_flash_cookies should only be called after anything that relies on the page existing in the browser.

**Comment by dreisman** [comment]
Made a few minor doc edits to emphasize the point, but as commands have more side effects we could also throw some sort of error if you add a command that requires the page after a command that ends the page visit.

**Comment by englehardt** [comment]
Thanks @dreisman! Added #99 as a follow up.

**Comment by johanmazel** [comment]
Thank you very much for finding my mistake.
Sorry for submitting an invalid issue.
